### DISCUSSION TITLE
Add list of transactions to Balances view

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#6209] [Ideas]
I'm imagining 

```
[    $40.00    |    $56.00   ]              <- the balances table
[    Ole $     |    Flex $   ]

[ 4 weekly meals remaining   ]   <- the meal plans
[ 2 daily meals remaining    ]

[     Ole     |     Flex     ]                   <- a toggle button
| Pause Micros         $2.40 |
| ғᴇʙʀᴜᴀʀʏ 14                |
| Cage Micros 3        $5.60 |
| ғᴇʙʀᴜᴀʀʏ 13                |
| Cage Micros 4        $5.60 |
| ғᴇʙʀᴜᴀʀʏ 12                |
```

It would only show recent transactions, at least in the first iteration, so I'm limiting it to ten rows. 

I'm envisioning a UIToggleButton to switch between funding sources, and I deliberately left off Printing from here – that data would come from a different source that I don't  necessarily have access to from the OleCard endpoint.

**Comment by drewvolz** [comment]
Could even get fancy with pie charts from Victory

**Comment by drewvolz** [comment]
I think the transaction history would be useful to display. Good thinking!

**Comment by hawkrives** [comment]
Let's push off pie charts to v2 of the transactions list, just for simplicity
